### PR TITLE
Bug 

### PR DESCRIPTION
**Pull request by sabry2020**

### CODE REVIEW

**Comment by sabry2020** [comment]
there is no file name formed  its just test

File: Recorder/Recorder.cs [hunk 1]
```diff
@@ -1,204 +1,208 @@
...
+            {
+                Save(fileName);
+            }
+        }
+
+        #endregion
+
+        #region Recorder Functions
+
+        public static void Save(string fileName)
```
Comment (ah1053) [inline Recorder/Recorder.cs]: fileName = "test" as the default value

File: Recorder/Recorder.cs [hunk 2]
```diff
@@ -1,204 +1,208 @@
...
+        {
+            audioSource = GetComponent<AudioSource>();
+            audioSource.clip = Microphone.Start(Microphone.devices[0], true, 10, 44100);  // third argument restrict the duration of the audio to 10 seconds 
+            if (SaveButton == null)
+            {
+                return;
+            }
+            SaveButton.onClick.AddListener(() =>
+            {
+                Save(fileName="test");
```
Comment (ah1053) [inline Recorder/Recorder.cs]: Save(fileName)

**Comment by ah1053** [comment]
Good Job, Thanks

**Pull request merged**